build-scripts/gulp/cast.js

```diff
@@ -13,7 +13,6 @@ gulp.task(
   gulp.series(
     async function setEnv() {
       process.env.NODE_ENV = "development";
-      process.env.IS_CAST = "true";
```

Comment:
Still used in `build-scripts/gulp/translations.js:279`

Reply:
I'm thinking now, I think that we can remove the old translations for cast now that we mix in backend. You can remove that whole hack.